File: notify/v1/segment_membership/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Notify.V1.Service.User;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var segmentMembership = SegmentMembershipResource.Fetch(
            pathServiceSid: "ISaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathIdentity: "NUaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathSegment: "PathSegment"
        );

        Console.WriteLine(segmentMembership.AccountSid);
    }
}

File: notify/v1/user_binding/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

user_binding = @client.notify
                      .services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                      .users('NUXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                      .bindings('BSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                      .fetch

puts user_binding.sid
